packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -30,6 +30,7 @@ def jasmine_node_test(
         tags = [],
         jasmine = Label("@npm//@bazel/jasmine"),
         coverage = False,
+        seed = None,
```

Comment:
this API seems reasonable to me.
I wonder if we should also expose whether or not to randomize tests

Reply:
Okay... I've added that feature as well.